OrbitGl/TimeGraph.cpp

```diff
@@ -31,7 +31,10 @@
 #include "TimerManager.h"
 #include "Utils.h"
 #include "absl/strings/str_format.h"
+#include "absl/flags/flag.h"
 
+// TODO: Remove this flag once we have a way to toggle the display return values
+ABSL_FLAG(bool, show_return_values, false, "Show return values on time slices");
```

Comment:
We have more and more flags scattered in different source files. I believe these should be moved to main.cpp. The rationale here is that when we have a way to toggle this from the user interface we want to have mechanism in place to set the flag for capturing without resorting to globals or ABSL_FLAG values.